blueprints/data-solutions/dataproc-lifecycle-via-composer/terraform/variables.tf

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project_id" {
+  description = "Project ID where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "pso-amex-data-platform"
+  type        = string
+}
+
+variable "region" {
+  description = "Region where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "us-central1"
+  type        = string
+}
+
+variable "zone" {
+  description = "Zone where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "us-central1-a"
+  type        = string
+}
+
+variable "network" {
+  description = "Network ID to be used by the Cloud Composer Environment, and the dataproc clusters, should exist in advance in the defined region and zone."
+  default     = "default"
+  type        = string
+}
+
+variable "subnetwork" {
+  description = "Subnetwork ID to be used by the Cloud Composer Environment, and the dataproc clusters, should exist in advance in the defined region, zone, and network."
+  default     = "default"
+  type        = string
+}
+
+variable "composer_env_name" {
+  description = "Name of Cloud Composer Environment to be created."
+  default     = "composer-env-dataproc-ej"
+  type        = string
+}
+
+variable "dataproc_lifecycle_dag" {
```

Comment:
can you bundle all dataproc variables into a `dataproc_config` variable?

Reply:
dataproc variables bundled into an dataproc config variable